java/component/keyple-calypso/src/main/java/org/eclipse/keyple/calypso/transaction/CalypsoPo.java

```diff
@@ -290,7 +290,7 @@ protected boolean isSerialNumberExpiring() {
      * 
      * @return a byte array containing the ATR (variable length)
      */
-    public byte[] getAtr() {
+    public byte[] getAtrBytes() {
```

Comment:
Should this method instead be inherited from AbstractMatchingSE?

Reply:
The CalypsoPo.poAtr attribute should be removed, this value is already present in AbstractMatchingSe.

Reply:
Good point!
It's a consequence of the changes in AbstractMatchingSe that I hadn't spotted.